e2e/playwright/native-file-menu.spec.ts

```diff
@@ -1,2375 +1,539 @@
+        await clickElectronNativeMenuById(tronApp, 'View.Command Palette...')
+        // Check the placeholder project name exists
+        const actual = cmdBar.cmdBarElement.getByTestId('cmd-bar-search')
+        await expect(actual).toBeVisible()
+      })
 
-          // Run electron snippet to find the Menu!
-          await page.waitForTimeout(100) // wait for createModelingPageMenu() to run
-          await tronApp.electron.evaluate(async ({ app }) => {
-            if (!app || !app.applicationMenu) {
-              throw new Error('app or app.applicationMenu is missing')
-            }
-            const menu = app.applicationMenu.getMenuItemById(
-              'View.Standard views.Back view'
-            )
-            if (!menu) {
-              throw new Error('menu missing')
-            }
-            // menu.click()
-          })
-        })
-        test('Modeling.View.Standard views.Top view', async ({
+      await test.step('Home.Help.Show all commands', async () => {
+        await page.reload()
+        await homePage.projectsLoaded()
+        await homePage.isNativeFileMenuCreated()
+        await clickElectronNativeMenuById(tronApp, 'Help.Show all commands')
+        // Check the placeholder project name exists
+        const actual = cmdBar.cmdBarElement.getByTestId('cmd-bar-search')
+        await expect(actual).toBeVisible()
+      })
+      await test.step('Home.Help.KCL code samples', async () => {
+        await page.reload()
+        await homePage.projectsLoaded()
+        await homePage.isNativeFileMenuCreated()
+        await clickElectronNativeMenuById(tronApp, 'Help.KCL code samples')
+      })
+      await test.step('Home.Help.Report a bug', async () => {
+        await page.reload()
+        await homePage.projectsLoaded()
+        await homePage.isNativeFileMenuCreated()
+        await clickElectronNativeMenuById(tronApp, 'Help.Report a bug')
+        // Core dump and refresh magic number timeout
+        await page.waitForTimeout(7000)
+        await homePage.projectsLoaded()
+      })
+      await test.step('Home.Help.Replay onboarding tutorial', async () => {
+        await page.reload()
+        await homePage.projectsLoaded()
+        await homePage.isNativeFileMenuCreated()
+        await findElectronNativeMenuById(
+          tronApp,
+          'Help.Replay onboarding tutorial'
+        )
+      })
+      await test.step('Home.File.Preferences.Sign out', async () => {
+        await page.reload()
+        await homePage.projectsLoaded()
+        await homePage.isNativeFileMenuCreated()
+        await clickElectronNativeMenuById(tronApp, 'File.Sign out')
+        // FIXME: When signing out during E2E the page is not bound correctly.
+        // It cannot find the button
+        // const signIn = page.getByTestId('sign-in-button')
+        // await expect(signIn).toBeVisible()
+      })
+    }
+  )
+  test(
+    'Modeling page all in one test but uses steps',
```

Comment:
Up to you, but I don't think any other tests mention that they include steps in their name. This feels unnecessary to me.

```suggestion
    'Modeling page',
```